libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/MapboxNativeNavigator.kt

```diff
@@ -58,6 +60,12 @@ interface MapboxNativeNavigator {
 
     suspend fun resetRideSession()
 
+    fun startNavigationSession()
+
+    fun stopNavigationSession()
+
+    fun restoreNavigationSession(): NavigationSessionState
```

Comment:
How store / restore parity is implemented?

Reply:
discussed, moved store/restore under impl